src/ui/Facet/ValueElement.ts

```diff
@@ -198,6 +198,12 @@ export class ValueElement {
     eventArg.clear();
   }
 
+  private tryDismissSearchResults() {
+    if (this.facet && this.facet.facetSearch && this.facet.facetSearch.dismissSearchResults) {
+      this.facet.facetSearch.dismissSearchResults();
+    }
```

Comment:
I feel this approach is not very encapsulated, where we are trying to adjust the behaviour of the facet search from the outside.

Would it work if we instead added a listener inside the facet search, such that when a select/exclude is detected, it hides the search results? Is there an event we can hook onto?

Reply:
There is no existing event, but I can add one.